en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md

```diff
@@ -0,0 +1,31 @@
+# Salesforce REST Connector Overview
+
+This documentation provides overall mapping with each sections in Salesforce connector and a inbound endpoint documentation. Click the links of the shown bellow to view the documentation for **Salesforce REST Connector** and **Salesforce Inbound Endpoint**.
```

Comment:
```suggestion
This documentation provides overall mapping with each sections in Salesforce connector and a inbound endpoint documentation. Click the links below to view the documentation for **Salesforce REST Connector** and **Salesforce Inbound Endpoint**.
```
Shall we replace the first sentence with some information about the following:
- API(s) that we are connecting to.
- The connectors we have.

Reply:
Updated.